cfpb/consumerfinance.gov
# Archive wagtail events cron job

Archives Event Pages in wagtail, see comments in cronjob for further explanation. Runs daily and new runs will replace the old via its default deletion policy
---

## Additions

- one cron job that runs the archive events command in Django-admin



## How to test this PR

1.update/deploy your  local helm instance
2.add an Events page that is scheduled to the past inthe about us section of the wagtail site
2. trigger the archived-events cronjob3. 
3. check the logs of it moving the new event to archive


  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
